src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -20,39 +20,41 @@
 import com.google.common.collect.Sets;
 import com.mojang.brigadier.StringReader;
 import com.mojang.brigadier.exceptions.CommandSyntaxException;
-import net.minecraft.entity.SpawnGroup;
-import net.minecraft.entity.ai.brain.Brain;
-import net.minecraft.entity.ai.brain.Memory;
-import net.minecraft.entity.EntityGroup;
-import net.minecraft.entity.ai.brain.MemoryModuleType;
-import net.minecraft.entity.ai.pathing.Path;
-import net.minecraft.entity.attribute.AttributeContainer;
-import net.minecraft.entity.attribute.EntityAttribute;
-import net.minecraft.entity.projectile.ProjectileEntity;
-import net.minecraft.entity.vehicle.AbstractMinecartEntity;
-import net.minecraft.nbt.NbtString;
-import net.minecraft.nbt.NbtElement;
-import net.minecraft.network.packet.s2c.play.EntityPassengersSetS2CPacket;
-import net.minecraft.network.packet.s2c.play.EntityPositionS2CPacket;
-import net.minecraft.network.packet.s2c.play.ExperienceBarUpdateS2CPacket;
-import net.minecraft.network.packet.s2c.play.PlayerPositionLookS2CPacket;
```

Comment:
I know its just imports, but refrain from resorting them mid double branch

Reply:
Uhm, what do you mean? I'm not using this imports in my PR

Reply:
This was from a previous commit.